<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assistive Navigation â€” GUI (Designer/Qt6 compatible)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainArea">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>921</width>
      <height>811</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <widget class="QLabel" name="videoMain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>450</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string>videoMain</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="videoRoad">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>421</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string>videoRoad</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGroupBox" name="connectionPanel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>431</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>connectionPanel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="logTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>881</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>logTable</string>
     </property>
    </widget>
    <widget class="QLabel" name="bhcLabel">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>360</y>
       <width>411</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="bhcLabel_2">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>V1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="bhcLabel_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>0</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>V2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
